src/main/java/marquez/api/models/DatasourceResponse.java

```diff
@@ -0,0 +1,21 @@
+package marquez.api.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Do we need 3 lombok annotations here? If we have `@Data`, what are the values of adding `@NoArgsConstructor` and `@AllArgsConstructor`?

Reply:
I was following the NamespaceResponse example, but agree that a cleanup to only have what's necessary is a good idea.

Reply:
Great. I opened an issue to capture the changes we'll need to make across API models: #354

Reply:
You're right - if we add the `@Getter` annotation to each member, then we can remove the `@Data` annotation.

That said, the approach advocated in #354 doesn't seem to work here. This might be b/c this is a Response object. However, I'll give it a try on `DatasourceRequest` when I add it.

If we remove the AllArgsConstructor, we get compilation issues when building the DatasourceResponse object. 

If we remove the NoArgsConstructor, we get this issue that we saw before:
```
Caused by: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Cannot construct instance of `marquez.api.models.DatasourceResponse` (no Creators, like default construct, exist): cannot deserialize from Object value (no delegate- or property-based Creator)
 at [Source: (org.glassfish.jersey.message.internal.ReaderInterceptorExecutor$UnCloseableInputStream); line: 1, column: 18] (through reference chain: marquez.api.models.DatasourcesResponse["datasources"]->java.util.ArrayList[0])
```

Reply:
Oh, sorry. I linked the wrong issue :P #355

Reply:
Please see PRs #394, #387, and #384, where this is fixed.